src/TabPane.js

```diff
@@ -265,12 +260,12 @@ class TabPane extends React.Component {
       return (
         <Transition
           in={active && !exiting}
-          onEnter={createChainedFunction(this.handleEnter, onEnter)}
+          onEnter={this.handleEnter}
```

Comment:
maybe add a comment in the spread indicating that those are deliberately unused, and are there to be omitted from `elementProps` above?